processors/pybossa_indications/processor.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from itertools import groupby
+
+import os
+import time
+import requests
+import pbclient as pbc
+import logging
+logger = logging.getLogger(__name__)
+
+PYBOSSA_URL = os.environ['PYBOSSA_URL']
+PYBOSSA_API_KEY = os.environ['PYBOSSA_API_KEY']
+PROJECT_ID = os.environ['PYBOSSA_PROJECT_INDICATIONS']
```

Comment:
Instead of hardcoding them here, add those on https://github.com/opentrials/processors/blob/master/processors/base/config.py and get this from the `conf` dict passed to `process()`